doc/devdocs/readme.md

```diff
@@ -17,6 +17,11 @@ Once you've discussed your proposed feature/fix/etc. with a team member, and you
 - Try to package new ideas/components into libraries that have nicely defined interfaces.
 - Package new ideas into classes or refactor existing ideas into a class as you extend.
 - When adding new classes/methods/changing existing code: add new unit tests or update the existing tests.
+- When adding new .Net projects enable the analyzer in `Recommended` mode. You have to add the following code lines in the first block of the `.csproj` file:
```

Comment:
My goal is to add this to the [Directory.Build.props](https://github.com/microsoft/PowerToys/blob/main/Directory.Build.props) when I work on #16423. It would then automatically applied to all projects.

Reply:
this an outdated commit